src/core/xref.js

```diff
@@ -961,6 +961,12 @@ class XRef {
         );
       }
       nums[i] = num;
+
+      const entry = this.getEntry(num);
```

Comment:
This new block could use a comment to explain its purpose for future readers (basically briefly indicating what we're fixing here, with a reference to bug 1978317).